packages/shared_preferences/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
-## NEXT
+## 2.2.0
 
 * Increase the minimum Flutter version to 3.3.
+* Update shared_preferences to 2.2.0.
+* Update shared_preferences_interface to 2.3.0.
+* Update integration_test.
```

Comment:
The point of this update is the addition of two APIs. It would be good to write this down as well. (clearWithParameters, getAllWithParameters)

Reply:
Added it!